packages/cfd/src/Containers/jurisdiction-modal-content.tsx

```diff
@@ -0,0 +1,823 @@
+            } else if (type_of_card === 'bvi' || type_of_card === 'labuan' || type_of_card === 'maltainvest') {
+                if (poi_acknowledged_for_bvi_labuan && poa_acknowledged) {
+                    return (
+                        <div className={`${card_classname}__verification-status`}>
+                            <div className={`${card_classname}__verification-status--pending`}>
+                                <Text size='xxxs' color={'prominent'}>
+                                    <Localize i18n_default_text='Pending verification' />
+                                </Text>
+                            </div>
+                        </div>
+                    );
+                } else if (need_poi_for_bvi_labuan && poa_acknowledged) {
+                    return (
+                        <div className={`${card_classname}__verification-status`}>
+                            <div className={`${card_classname}__verification-status--POA_POI`}>
+                                <Text size='xxxs' color={'white'}>
+                                    <Localize i18n_default_text='Check your proof of identity' />
+                                </Text>
+                            </div>
+                        </div>
+                    );
+                } else if (need_poi_for_bvi_labuan && !poa_acknowledged) {
+                    return (
+                        <div className={`${card_classname}__verification-status`}>
+                            <div className={`${card_classname}__verification-status--POA_POI`}>
+                                <Text size='xxxs' color={'white'}>
+                                    <Localize i18n_default_text='Check your proof of identity and address' />
+                                </Text>
+                            </div>
+                        </div>
+                    );
+                }
+            }
+            return null;
+        }
+        // account added
+        return (
+            <div className={`${card_classname}__verification-status`}>
+                <div className={`${card_classname}__verification-status--verified`}>
+                    <Text size='xxxs' className={`${card_classname}__verification-status--verified-text`} weight='bold'>
+                        <Localize i18n_default_text='Account added' />
+                    </Text>
+                </div>
+            </div>
+        );
+    };
+
+    return (
+        <>
+            <div
+                className={classNames(card_classname, {
+                    [`${card_classname}--selected`]: jurisdiction_selected_shortcode === type_of_card,
+                })}
+                onClick={disabled ? () => undefined : () => cardSelection(`${type_of_card}`)}
+                style={one_or_two_cards ? { width: '32em' } : { width: '27.6em' }}
+            >
+                {jurisdiction_contents[type_of_card as keyof typeof jurisdiction_contents].is_over_header_available && (
```

Comment:
```suggestion
    const the_thing = React.useMemo(() => {
       return jurisdiction_contents[type_of_card as keyof typeof 
    jurisdiction_contents]}, [type_of_card]);
```
and use it where it's needed.